src/components/dialogs/parameters/dynamicsimulation/curve/dialog/equipment-filter.jsx

```diff
@@ -167,28 +165,25 @@ const EquipmentFilter = forwardRef(
         const [selectedVoltageLevelIds, setSelectedVoltageLevelIds] = useState(
             []
         );
-        const handleVoltageLevelChange = useCallback(
-            (selectedVoltageLevelIds) => {
-                setSelectedVoltageLevelIds(selectedVoltageLevelIds);
-            },
-            []
-        );
+        const handleVoltageLevelChange = (selectedVoltageLevelIds) => {
```

Comment:
No ! It's worse than before.
Just give directly the setter and don't create a new function.

Reply:
DONE